ev3dev/core.py

```diff
@@ -958,93 +962,84 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
-    def set_power(self, power):
-        self.speed_sp = power
-
-    def set_position_rotations(self, power, rotations):
+    def _set_position_rotations(self, power, rotations):
         if power > 0:
             self.position_sp = self.position + int(rotations * self.count_per_rot)
         else:
             self.position_sp = self.position - int(rotations * self.count_per_rot)
 
-    def set_position_degrees(self, power, degrees):
+    def _set_position_degrees(self, power, degrees):
         rotations = float(degrees / self.count_per_rot)
-        self.set_position_rotations(power, rotations)
+        self._set_position_rotations(power, rotations)
 
-    def set_brake(self, brake):
+    def _set_brake(self, brake):
         if brake:
-            self.stop_action = self.STOP_ACTION_BRAKE
+            self.stop_action = self.STOP_ACTION_HOLD
         else:
             self.stop_action = self.STOP_ACTION_COAST
 
-    def set_time(self, ms):
-        self.time_sp = ms
-
-    def on_for_rotations(self, power, rotations, brake):
+    def on_for_rotations(self, power, rotations, brake=True, block=True):
```

Comment:
What do others think about calling it "speed" instead of "power"? This would only apply if we switched to using scientific units.

Reply:
yeah the use of `power` is some places and `speed` in other is already making my head spin :)  I think the EV3-G blocks use `power`?